lib/Echidna/Solidity.hs

```diff
@@ -156,7 +161,7 @@ extractConstants :: [SolcContract] -> [AbiValue]
 extractConstants = nub . concatMap (constants "" . view contractAst) where
   -- Tools for parsing numbers and quoted strings from 'Text'
   as f     = preview $ to f . _Right . _1
-  asAddr x = as hexadecimal =<< T.stripPrefix "0x" x
+  --asAddr x = as hexadecimal =<< T.stripPrefix "0x" x
```

Comment:
why did we comment this/the below out?

Reply:
Since we use `crytic-compile` for this, we are forced to use the [`compact-format` AST](https://github.com/crytic/echidna/pull/204/files#diff-d38ab0de3d2db1aebda046d6d01dbc64L90), where addresses are no longer especial values, but integers. That's why this is no longer necessary. So, I will remove this commented function.